.rubocop.yml

```diff
@@ -18,4 +18,7 @@ Metrics/LineLength:
   Enabled: true
 
 Metrics/LineLength:
-  Max: 120
\ No newline at end of file
+  Max: 120
+  
+Style/ClassAndModuleChildren:
+  Enabled: false
```

Comment:
The small symbol at the end of this line means that there is no "newline at the end of the file" - it's common best-practice to ensure that there's always one blank line at the end of every file as that will cut down on git "diff noise" or changes in the git diff that aren't real, like in this case it thinks line 21 was modified when it actually wasn't.

Sublime can actually take care of this for you automatically - you just have to go to Preferences (Command-comma on Mac, or in the menu on Mac or Windows) and ensure it has the following two settings set:

```json
{
	"ensure_newline_at_eof_on_save": true,
	"trim_trailing_white_space_on_save": true,
}
```

That will remove any extra spaces at the end of any line, as well as ensure that there's a blank line at the end of every file on save.  Atom and other editors do this out of the box, but with Sublime you have to turn it on.

Anyways, it doesn't need to be changed for this PR to merge, just a note for the future.